recipe/meta.yaml

```
@@ -23,19 +25,24 @@ requirements:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - appnope
-  requires:
-    - pip
   commands:
     - pip check
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - pytest -v tests
+  requires:
+    - pip
+    - pytest
 
 about:
   home: https://github.com/minrk/appnope
+  summary: Disable App Nap on OS X 10.9
   license: BSD-2-Clause
   license_family: BSD
   license_file: LICENSE
-  summary: Disable App Nap on OS X 10.9
   description: |
     appnope is a simple package for disabling App Nap on OS X 10.9, which can be problematic.
```

```suggestion
    appnope is a simple package for disabling App Nap on OS X >=10.9, which can be problematic.
```
see https://github.com/minrk/appnope/tree/0.1.4?tab=readme-ov-file#appnope